packages/zosmf/__tests__/__snapshots__/ZosmfSession.unit.test.ts.snap

```diff
@@ -32,3 +32,19 @@ Object {
   "user": "fake",
 }
 `;
+
+exports[`zosmf utils should create a session object with timeout 1`] = `
+Object {
```

Comment:
Timeout shouldn't actually be on the zosmf session anymore, since the implementation changed. The test that created that snapshot has been removed and the snapshot has been updated.